pkg/async/redis/engine_test.go

```diff
@@ -436,7 +437,7 @@ func TestRunRespondsToCanceledContext(t *testing.T) {
 // amend these overrides to test specific scenarios.
 func getTestEngine() *engine {
 	config := NewConfigWithDefaults()
-	config.RedisHost = "redis"
+	config.RedisHost = os.Getenv("ASYNC_REDIS_HOST")
```

Comment:
See [this comment](https://github.com/Azure/open-service-broker-azure/pull/369/files#r177846967) for context.